web/src/settings_ui.ts

```diff
@@ -76,11 +93,11 @@ export function do_settings_change(
 // * disable_on_uncheck is boolean, true if sub setting should be disabled
 //   when main setting unchecked.
 export function disable_sub_setting_onchange(
-    is_checked,
-    sub_setting_id,
-    disable_on_uncheck,
-    include_label,
-) {
+    is_checked: boolean,
+    sub_setting_id: string,
+    disable_on_uncheck: boolean,
+    include_label?: boolean,
```

Comment:
A possibly-undefined boolean is almost never a good type to have. This should be a boolean that defaults to `false`.

Reply:
@andersk Okay, i'll make it `include_label: boolean = false`